spec/conflict_free_schema/database/schema_version_files_spec.rb

```diff
@@ -54,6 +56,61 @@
         end
       end
     end
+
+    context 'when "db/migrate" folder contains other subfolders' do
```

Comment:
as those tests were stolen (d'oh) from gitlab directly - I think that maybe I will extract new unit tests to a separate spec file

Ideally this lib should be updated according [to this suggestion](https://github.com/stolen-ruby/conflict_free_schema/issues/2#issuecomment-875130221) so we can have support for multiple databases - keeping unit tests in sync (with gitlab) will probably make upgrade process smoother 🤔

Reply:
At the moment in the GitLab repo "Conflict-free schema version handling" feature does not support the recursive search for migrations within `db/migration`.

So this library will be slightly better than the implementation in GitLab.

Reply:
I suppose it depends on one's needs 😉 

Thanks again for the contribution, will merge and refer to rest of my comments directly on main branch